velodyne_description/urdf/example.urdf.xacro

```diff
@@ -37,9 +37,14 @@
     <origin xyz="0 0 0.4" rpy="0 0 0" />
   </VLP-16>
 
-  <xacro:include filename="$(find velodyne_description)/urdf/HDL-32E.urdf.xacro"/>
-  <HDL-32E parent="base_link" name="velodyne2" topic="/velodyne_points2" hz="10" samples="220" gpu="${gpu}">
+  <xacro:include filename="$(find velodyne_description)/urdf/HDL-32E.urdf.xacro"/> -->
```

Comment:
Seems like there's an extra `-->` here...

Reply:
Oops, cleaned it up in https://github.com/ToyotaResearchInstitute/velodyne_simulator/pull/1/commits/894b94a1e949da080717fc062a20258299153a4f.